websites/C/Claude/metadata.json

```
@@ -0,0 +1,39 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.12",
+	"apiVersion": 1,
+	"author": {
+		"id": "1180088441459978261",
+		"name": "KohnoseLami"
+	},
+	"contributors": [],
+	"service": "Claude",
+	"description": {
+		"en": "Claude is a family of large language models developed by Anthropic. The first model was released in March 2023.\n\nThe Claude 3 family, released in March 2024, consists of three models: Haiku optimized for speed, Sonnet balancing capabilities and performance, and Opus designed for complex reasoning tasks. These models can process both text and images, with Claude 3 Opus demonstrating enhanced capabilities in areas like mathematics, programming, and logical reasoning compared to previous versions.",
+		"ja_JP": "Claude（クロード）とは、Anthropic社が開発した大規模言語モデルを用いた対話型生成AIである。2023年3月14日に一般公開され、以降複数のバージョンがリリースされている。\n\n2024年3月に公開されたClaude 3 OpusはメンサのIQテストでIQスコア101を記録した。「フレンドリーで熱心な同僚」をコンセプトの一つとしている。"
+	},
+	"url": "claude.ai",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/SqUGiUx.png",
```

Transparent:
```suggestion
	"logo": "https://i.imgur.com/6hBiEd2.png",
```

Non transparent:
```suggestion
	"logo": "https://i.imgur.com/t2w5Nso.png",
```